### PR TITLE
Sasho9203 create entity category for database

### PR DESCRIPTION
**Pull request by OlexKov**

### CODE REVIEW

File: data_access/Data/DefaultData.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+namespace data_access.Data
+{
+    internal static class DefaultData
+    {
+
+        public static void Initialize(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<User>().HasData(Users);
+            modelBuilder.Entity<Category>().HasData(Categories);
+
```
Comment (vladtymo) [inline data_access/Data/DefaultData.cs]: Remove empty line

File: data_access/Data/DefaultData.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        {
+            new User(){ Id = 1, NikName = "Fagot" , PasswordHash = "19513fdc9da4fb72a4a05eb66917548d3c90ff94d5419e1f2363eea89dfee1dd"},     //Password1
+            new User(){ Id = 2, NikName = "Veronica" , PasswordHash = "1be0222750aaf3889ab95b5d593ba12e4ff1046474702d6b4779f4b527305b23"},  //Password2
+            new User(){ Id = 3, NikName = "Alex" , PasswordHash = "2538f153f36161c45c3c90afaa3f9ccc5b0fa5554c7c582efe67193abb2d5202"}       //Password3
+
+        };
+
+        public static readonly Category[] Categories =
+        {
+
```
Comment (vladtymo) [inline data_access/Data/DefaultData.cs:29]: Add init data

File: data_access/Entities/Category.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+using System.Threading.Tasks;
+
+namespace data_access.Entities
+{
+    public class Category
+    {
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+    }
```
Comment (vladtymo) [inline data_access/Entities/Category.cs:8]: Add navigation properties

File: data_access/Entities/Category.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (vladtymo) [inline data_access/Entities/Category.cs]: Remove unnecessary usings

**Review by vladtymo — CHANGES_REQUESTED** [review]

**Review by Kllsel — APPROVED** [review]

**Review by vladtymo — APPROVED** [review]

**Pull request merged**